### ISSUE TITLE
main not detected in MIPS

### ISSUE THREAD

**Issue by crowell** [radare/radare2#2074]
```sh
ikana ~ » objdump -D ./a.out|grep main
004006f0 <main>:
ikana ~ » r2 ./a.out
 -- Wait a moment..
[0x00400570]> s main
[0x00000000]> aa
[0x00000000]> s main
[0x00000000]> q
ikana ~ » 
```

base64 of the binary

```
f0VMRgEBAQAAAAAAAAAAAAIACAABAAAAcAVAADQAAAB8EgAABxAAEDQAIAAKACgAIQAeAAYAAAA0
AAAANABAADQAQABAAQAAQAEAAAUAAAAEAAAAAwAAAHQBAAB0AUAAdAFAAA0AAAANAAAABAAAAAEA
AAADAABwqAEAAKgBQACoAUAAGAAAABgAAAAEAAAACAAAAAAAAHDAAQAAwAFAAMABQAAYAAAAGAAA
AAQAAAAEAAAAAQAAAAAAAAAAAEAAAABAAHQIAAB0CAAABQAAAAAAAQABAAAAdAgAAHQIQQB0CEEA
aAAAAHwAAAAGAAAAAAABAAIAAAD8AQAA/AFAAPwBQAD4AAAA+AAAAAQAAAAEAAAABAAAAIQBAACE
AUAAhAFAACAAAAAgAAAABAAAAAQAAAAEAAAA2AEAANgBQADYAUAAJAAAACQAAAAEAAAABAAAAAAA
AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAEAAAAL2xpYi9sZC5zby4xAAAAAAQAAAAQAAAAAQAA
AEdOVQAAAAAAAgAAAAYAAAAgAAAAAAAAAAAAAgABAQABAAAAAAAAAAAAAAAAAAAAAPYBALIAAAAA
AAAAAAAAAAAAAAAAkIhBAAQAAAAUAAAAAwAAAEdOVQDZbODLROvJ+HR/MQMxW03aiXrkVgEAAACR
AAAADAAAACwFQAANAAAAIAhAABkAAAB0CEEAGwAAAAQAAAAaAAAAeAhBABwAAAAEAAAABAAAAPQC
QAAFAAAAIARAAAYAAABAA0AACgAAAM8AAAALAAAAEAAAABYAAHCQCEEAFQAAAAAAAAADAAAAoAhB
AAEAAHABAAAABQAAcAIAAAAGAABwAABAAAoAAHAIAAAAEQAAcA4AAAASAABwHgAAABMAAHAIAAAA
/v//bwwFQAD///9vAQAAAPD//2/wBEAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
AAAAAAAAAAAAAAAAAAAAAwAAAA4AAAAHAAAACAAAAAQAAAAAAAAAAAAAAAAAAAAAAAAAAwAAAAoA
AAAFAAAACQAAAAYAAAALAAAADAAAAAEAAAANAAAAAgAAAAAAAAAAAAAAAAAAAAAAAAARAAAA5AdA
AAgAAAASAA0AmwAAAAEAAAAAAAAAEwDx/7YAAABQCEAABAAAABEAEACsAAAAkAhBAAAAAAARABYA
AQAAAEAHQACkAAAAEgANAC4AAADwBkAAUAAAABIADQALAAAALAVAAAAAAAASAAwAXgAAAAAAAAAA
AAAAIAAAAHgAAAAAAAAAAAAAACAAAAAhAAAAAAhAAAAAAAASAAAAMwAAAAAAAAAAAAAAIgAAAIwA
AADwB0AAAAAAABIAAABCAAAAAAAAAAAAAAAgAAAAAF9fbGliY19jc3VfaW5pdABfX2xpYmNfY3N1
X2ZpbmkAX19saWJjX3N0YXJ0X21haW4AX19nbW9uX3N0YXJ0X18AX0lUTV9kZXJlZ2lzdGVyVE1D
bG9uZVRhYmxlAF9JVE1fcmVnaXN0ZXJUTUNsb25lVGFibGUAX0p2X1JlZ2lzdGVyQ2xhc3NlcwBw
dXRzAGxpYmMuc28uNgBfRFlOQU1JQ19MSU5LSU5HAF9fUkxEX01BUABfSU9fc3RkaW5fdXNlZABH
TElCQ18yLjAAAAAAAQABAAEAAQABAAEAAQAAAAAAAgAAAAIAAAABAAEAkQAAABAAAAAAAAAAEGlp
DQAAAgDFAAAAAAAAAAIAHDxkg5wnIeCZA+D/vScQALyvHAC/rzyAgo8EAEAQJQggADyAmY8J+CAD
JQggABwAv48IAOADIAC9JwAAAAAAAAAAIQDgAwEAEQQlCCAAQgAcPJCInCch+AAAGICEjwAApY8E
AKYn+P8BJCTooQPg/70nHICHjyCAiI8QAKivFACirxgAva84gJmPCfggAyUIIAD//wAQJQggACUI
IAAlCCAAQQAEPEEAAjyUCIQklwhCJCMQRAAHAEIsBwBAFEIAHDyQiJwnRICZjwMAIBMlCCAACAAg
AyUIIAAIAOADJQggAEEABDxBAAI8lAiEJJQIQiQjEEQAgxACAMIvAgAhKKIAQygFAAcAoBBCABw8
kIicJzCAmY8DACATJQggAAgAIAMlCCAACADgAyUIIADg/70nGACwr0EAEDzgCAKSBQBAFBwAv690
ARAMJQggAAEAAiTgCAKiHAC/jxgAsI8IAOADIAC9J0EABDx8CIQkAACCjAMAQBRCABw8hAEQCCUI
IADg/70nkIicJzSAmY8cAL+vEAC8rwUAIBMcAL+PCfggAyUIIAAQALyPHAC/j4QBEAggAL0nJQgg
ACUIIAAlCCAA4P+9JxwAv68YAL6vIfCgA0IAHDyQiJwnEAC8r0AAAjxgCEQkQICCjyHIQAAJ+CAD
JQggABAA3I8h6MADHAC/jxgAvo8gAL0nCADgAyUIIAACABw8UIGcJyHgmQPI/70nJICZjxAAvK8w
ALWvLAC0rygAs68kALKvIACxrzQAv68cALCvIZiAACGgoAAJ+CADIajAABAAvI8ogJGPLICSjyOQ
UQKDkBIACQBAEiGAAAAAADmOISBgAiEogAIhMKACCfggAwEAECb5/xIWBAAxJjQAv48wALWPLAC0
jygAs48kALKPIACxjxwAsI8IAOADOAC9JwgA4AMlCCAAJQggABCAmY8heOADCfggAwwAGCQQgJmP
IXjgAwn4IAMKABgkAAAAAAAAAAAAAAAAAAAAAAIAHDxwgJwnIeCZA+D/vScQALyvHAC/rxwAv48I
AOADIAC9JwAAAAAAAAAAAAAAAAEAAgAAAAAAAAAAAAAAAABoaQAAAAAAAAAAAAAAAAAAAAAAAJQG
QABcBkAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAIDwBkAAQAdA
AOQHQAAsBUAAdAhBAHgIQQAAAAAAAAAAAAAIQAAAAAAA8AdAAAAAAAAAAAAAR0NDOiAoRGViaWFu
IDQuOS4xLTE5KSA0LjkuMQBHQ0M6IChEZWJpYW4gNC44LjMtMTMpIDQuOC4zAAAA8AZAAAAAAMD8
////AAAAAAAAAAAgAAAAHgAAAB8AAABAB0AAAAA/gPz///8AAAAAAAAAADgAAAAdAAAAHwAAAOQH
QAAAAAAAAAAAAAAAAAAAAAAAAAAAAB0AAAAfAAAAQQ8AAABnbnUAAQcAAAAEAQAuc3ltdGFiAC5z
dHJ0YWIALnNoc3RydGFiAC5pbnRlcnAALm5vdGUuQUJJLXRhZwAuTUlQUy5hYmlmbGFncwAucmVn
aW5mbwAubm90ZS5nbnUuYnVpbGQtaWQALmR5bmFtaWMALmhhc2gALmR5bnN5bQAuZHluc3RyAC5n
bnUudmVyc2lvbgAuZ251LnZlcnNpb25fcgAuaW5pdAAudGV4dAAuTUlQUy5zdHVicwAuZmluaQAu
cm9kYXRhAC5laF9mcmFtZQAuaW5pdF9hcnJheQAuZmluaV9hcnJheQAuamNyAC5kYXRhAC5ybGRf
bWFwAC5nb3QALnNkYXRhAC5ic3MALmNvbW1lbnQALnBkcgAuZ251LmF0dHJpYnV0ZXMALm1kZWJ1
Zy5hYmkzMgAAAAAAAAAAAAAAAAAAAAAAAAAAAHQBQAAAAAAAAwABAAAAAACEAUAAAAAAAAMAAgAA
AAAAqAFAAAAAAAADAAMAAAAAAMABQAAAAAAAAwAEAAAAAADYAUAAAAAAAAMABQAAAAAA/AFAAAAA
AAADAAYAAAAAAPQCQAAAAAAAAwAHAAAAAABAA0AAAAAAAAMACAAAAAAAIARAAAAAAAADAAkAAAAA
APAEQAAAAAAAAwAKAAAAAAAMBUAAAAAAAAMACwAAAAAALAVAAAAAAAADAAwAAAAAAHAFQAAAAAAA
AwANAAAAAADwB0AAAAAAAAMADgAAAAAAIAhAAAAAAAADAA8AAAAAAFAIQAAAAAAAAwAQAAAAAABw
CEAAAAAAAAMAEQAAAAAAdAhBAAAAAAADABIAAAAAAHgIQQAAAAAAAwATAAAAAAB8CEEAAAAAAAMA
FAAAAAAAgAhBAAAAAAADABUAAAAAAJAIQQAAAAAAAwAWAAAAAACgCEEAAAAAAAMAFwAAAAAA2AhB
AAAAAAADABgAAAAAAOAIQQAAAAAAAwAZAAAAAAAAAAAAAAAAAAMAGgAAAAAAAAAAAAAAAAADABsA
AAAAAAAAAAAAAAAAAwAcAAAAAAAAAAAAAAAAAAMAHQABAAAAAAAAAAAAAAAEAPH/RAAAAMAFQAAA
AAAAAAANAEgAAAAAAAAAAAAAAAQA8f9TAAAAfAhBAAAAAAABABQAYAAAANAFQAAAAAAAAgANAHUA
AAAQBkAAAAAAAAIADQCIAAAAXAZAAAAAAAACAA0AngAAAOAIQQABAAAAAQAZAK0AAAB4CEEAAAAA
AAEAEwDUAAAAlAZAAAAAAAACAA0A4AAAAHQIQQAAAAAAAQASAP8AAAAAAAAAAAAAAAQA8f9IAAAA
AAAAAAAAAAAEAPH/BgEAAHAIQAAAAAAAAQARABQBAAB8CEEAAAAAAAEAFAAAAAAAAAAAAAAAAAAE
APH/IAEAAPAHQAAAAAAAAgAOAC0BAAB4CEEAAAAAAAAAEgA+AQAA/AFAAAAAAAABAAYARwEAAHQI
QQAAAAAAAAASAFoBAACQiEEAAAAAAAAA8f9eAQAA5AdAAAgAAAASAA0AbgEAAAEAAAAAAAAAEwDx
/38BAAAAAAAAAAAAACAAAACbAQAAgAhBAAAAAAAhABUApgEAANwIQQAAAAAAEAAYAK0BAAAgCEAA
AAAAABIADwCzAQAAgAhBAAAAAAARABUAwAEAAPAHQAAAAAAAEgAAANABAAAAAAAAAAAAACIAAADf
AQAA2AhBAAAAAAARAhgA7AEAAFAIQAAEAAAAEQAQAPsBAACACEEAAAAAABAAFQACAgAAAAhAAAAA
AAASAAAAHwIAAJAIQQAAAAAAEQAWACkCAABAB0AApAAAABIADQA5AgAA8AhBAAAAAAAQABkAPgIA
ANwIQQAAAAAAEAAZAEoCAADwBkAAUAAAABIADQBPAgAAAAAAAAAAAAARAAAAWAIAAHAFQAAAAAAA
EgANAGACAAAAAAAAAAAAABEAAABvAgAAcAVAAAAAAAAQAA0AdgIAAAAAAAAAAAAAIAAAAIoCAACU
CEEAAAAAABECFgCWAgAAoAhBAAAAAAARAhcArAIAANwIQQAAAAAAEAAZALICAAAAAAAAAAAAACAA
AADMAgAALAVAAAAAAAASAAwAAC91c3IvbGliL2djYy9taXBzZWwtbGludXgtZ251LzQuOS8uLi8u
Li8uLi9taXBzZWwtbGludXgtZ251L2NydDEubwBobHQAY3J0c3R1ZmYuYwBfX0pDUl9MSVNUX18A
ZGVyZWdpc3Rlcl90bV9jbG9uZXMAcmVnaXN0ZXJfdG1fY2xvbmVzAF9fZG9fZ2xvYmFsX2R0b3Jz
X2F1eABjb21wbGV0ZWQuNTk3NwBfX2RvX2dsb2JhbF9kdG9yc19hdXhfZmluaV9hcnJheV9lbnRy
eQBmcmFtZV9kdW1teQBfX2ZyYW1lX2R1bW15X2luaXRfYXJyYXlfZW50cnkAdGVzdC5jAF9fRlJB
TUVfRU5EX18AX19KQ1JfRU5EX18AX01JUFNfU1RVQlNfAF9faW5pdF9hcnJheV9lbmQAX0RZTkFN
SUMAX19pbml0X2FycmF5X3N0YXJ0AF9ncABfX2xpYmNfY3N1X2ZpbmkAX0RZTkFNSUNfTElOS0lO
RwBfSVRNX2RlcmVnaXN0ZXJUTUNsb25lVGFibGUAZGF0YV9zdGFydABfZWRhdGEAX2ZpbmkAX19k
YXRhX3N0YXJ0AHB1dHNAQEdMSUJDXzIuMABfX2dtb25fc3RhcnRfXwBfX2Rzb19oYW5kbGUAX0lP
X3N0ZGluX3VzZWQAX2ZkYXRhAF9fbGliY19zdGFydF9tYWluQEBHTElCQ18yLjAAX19STERfTUFQ
AF9fbGliY19jc3VfaW5pdABfZW5kAF9fYnNzX3N0YXJ0AG1haW4AX2dwX2Rpc3AAX19zdGFydABf
X2dudV9sb2NhbF9ncABfZnRleHQAX0p2X1JlZ2lzdGVyQ2xhc3NlcwBfX1RNQ19FTkRfXwBfR0xP
QkFMX09GRlNFVF9UQUJMRV8AX2Zic3MAX0lUTV9yZWdpc3RlclRNQ2xvbmVUYWJsZQBfaW5pdAAA
AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAbAAAAAQAAAAIAAAB0AUAA
dAEAAA0AAAAAAAAAAAAAAAEAAAAAAAAAIwAAAAcAAAACAAAAhAFAAIQBAAAgAAAAAAAAAAAAAAAE
AAAAAAAAADEAAAAqAABwAgAAAKgBQACoAQAAGAAAAAAAAAAAAAAACAAAABgAAABAAAAABgAAcAIA
AADAAUAAwAEAABgAAAAAAAAAAAAAAAQAAAAYAAAASQAAAAcAAAACAAAA2AFAANgBAAAkAAAAAAAA
AAAAAAAEAAAAAAAAAFwAAAAGAAAAAgAAAPwBQAD8AQAA+AAAAAkAAAAAAAAABAAAAAgAAABlAAAA
BQAAAAIAAAD0AkAA9AIAAEwAAAAIAAAAAAAAAAQAAAAEAAAAawAAAAsAAAACAAAAQANAAEADAADg
AAAACQAAAAEAAAAEAAAAEAAAAHMAAAADAAAAAgAAACAEQAAgBAAAzwAAAAAAAAAAAAAAAQAAAAAA
AAB7AAAA////bwIAAADwBEAA8AQAABwAAAAIAAAAAAAAAAIAAAACAAAAiAAAAP7//28CAAAADAVA
AAwFAAAgAAAACQAAAAEAAAAEAAAAAAAAAJcAAAABAAAABgAAACwFQAAsBQAAPAAAAAAAAAAAAAAA
BAAAAAAAAACdAAAAAQAAAAYAAABwBUAAcAUAAIACAAAAAAAAAAAAABAAAAAAAAAAowAAAAEAAAAG
AAAA8AdAAPAHAAAwAAAAAAAAAAAAAAAEAAAAAAAAAK8AAAABAAAABgAAACAIQAAgCAAAJAAAAAAA
AAAAAAAABAAAAAAAAAC1AAAAAQAAAAIAAABQCEAAUAgAACAAAAAAAAAAAAAAABAAAAAAAAAAvQAA
AAEAAAACAAAAcAhAAHAIAAAEAAAAAAAAAAAAAAAEAAAAAAAAAMcAAAAOAAAAAwAAAHQIQQB0CAAA
BAAAAAAAAAAAAAAABAAAAAAAAADTAAAADwAAAAMAAAB4CEEAeAgAAAQAAAAAAAAAAAAAAAQAAAAA
AAAA3wAAAAEAAAADAAAAfAhBAHwIAAAEAAAAAAAAAAAAAAAEAAAAAAAAAOQAAAABAAAAAwAAAIAI
QQCACAAAEAAAAAAAAAAAAAAAEAAAAAAAAADqAAAAAQAAAAMAAACQCEEAkAgAAAQAAAAAAAAAAAAA
AAQAAAAAAAAA8wAAAAEAAAADAAAQoAhBAKAIAAA4AAAAAAAAAAAAAAAQAAAABAAAAPgAAAABAAAA
AwAAENgIQQDYCAAABAAAAAAAAAAAAAAABAAAAAAAAAD/AAAACAAAAAMAAADgCEEA3AgAABAAAAAA
AAAAAAAAABAAAAAAAAAABAEAAAEAAAAwAAAAAAAAANwIAAA6AAAAAAAAAAAAAAABAAAAAQAAAA0B
AAABAAAAAAAAAAAAAAAYCQAAYAAAAAAAAAAAAAAABAAAAAAAAAASAQAA9f//bwAAAAAAAAAAeAkA
ABAAAAAAAAAAAAAAAAEAAAAAAAAAIgEAAAEAAAAAAAAAAAAAAIgJAAAAAAAAAAAAAAAAAAABAAAA
AAAAABEAAAADAAAAAAAAAAAAAACICQAAMAEAAAAAAAAAAAAAAQAAAAAAAAABAAAAAgAAAAAAAAAA
AAAAuAoAAPAEAAAgAAAAMwAAAAQAAAAQAAAACQAAAAMAAAAAAAAAAAAAAKgPAADSAgAAAAAAAAAA
AAABAAAAAAAAAA==
```

**Comment by radare** [comment]
Use sym.main, the main issue with mips analysis is that you need to know the value of %gp

**Comment by radare** [comment]
Fixed in 8a6e613ee17b8fb96a94e2599a0107c8bfde66ec
